weblate/glossary/models.py

```diff
@@ -204,7 +204,11 @@ def cleanup(text):
         """
         text = text.translate(CONTROLCHARS_TRANS).strip()
         if text and text[0] in PROHIBITED_INITIAL_CHARS:
-            text = text.lstrip("".join(PROHIBITED_INITIAL_CHARS))
+            text = text.lstrip(
+                "".join(
+                    PROHIBITED_INITIAL_CHARS | {" "},
```

Comment:
DeepL talks about "Unicode whitespace character", so that probably means any whitespace, not just the one you've added here.